app/assets/javascripts/discourse/tests/integration/components/user-menu/notifications-list-test.js

```diff
@@ -126,5 +125,114 @@ module(
+              read: false,
+              high_priority: false,
+              created_at: "2021-11-25T19:31:13.241Z",
+              post_number: 6,
+              topic_id: 10,
+              fancy_title: "Unread notification #01",
+              slug: "unread-notification-01",
+              data: {
+                topic_title: "Unread notification #01",
+                original_post_id: 20,
+                original_post_type: 1,
+                original_username: "discobot",
+                revision_number: null,
+                display_username: "discobot",
+              },
+            },
+            {
+              id: 13,
+              user_id: 1,
+              notification_type: NOTIFICATION_TYPES.replied,
+              read: false,
+              high_priority: false,
+              created_at: "2021-08-25T19:31:13.241Z",
+              post_number: 6,
+              topic_id: 10,
+              fancy_title: "Unread notification #02",
+              slug: "unread-notification-02",
+              data: {
+                topic_title: "Unread notification #02",
+                original_post_id: 20,
+                original_post_type: 1,
+                original_username: "discobot",
+                revision_number: null,
+                display_username: "discobot",
+              },
+            },
+            {
+              id: 81,
+              user_id: 1,
+              notification_type: NOTIFICATION_TYPES.mentioned,
+              read: true,
+              high_priority: false,
+              created_at: "2022-10-25T19:31:13.241Z",
+              post_number: 6,
+              topic_id: 10,
+              fancy_title: "Read notification #01",
+              slug: "read-notification-01",
+              data: {
+                topic_title: "Read notification #01",
+                original_post_id: 20,
+                original_post_type: 1,
+                original_username: "discobot",
+                revision_number: null,
+                display_username: "discobot",
+              },
+            },
+          ],
+          pending_reviewables: [
+            {
+              flagger_username: "sayo2",
+              id: 83,
+              pending: true,
+              topic_fancy_title: "anything hello world 0011",
+              type: "ReviewableQueuedPost",
+              created_at: "2022-09-25T19:31:13.241Z",
+            },
+            {
+              flagger_username: "sayo2",
+              id: 78,
+              pending: true,
+              topic_fancy_title: "anything hello world 0033",
+              type: "ReviewableQueuedPost",
+              created_at: "2021-06-25T19:31:13.241Z",
+            },
+          ],
+        });
+      });
+      await render(template);
+      const items = queryAll("ul li");
+      assert.ok(
+        items[0].textContent.includes("hello world 0011"),
+        "pending reviewables and unread notifications are sorted according to their created_at timestamp"
```

Comment:
This message is too generic for the assertion. When writing the messages for assertion, we want to describe the expected outcome only for that particular assertion. In this case, we might describe it as "unread notification is displayed first in the list"

Reply:
I've updated the assertion description. It's a bit verbose now, but I think it's very clear now what the expected outcome is.